### DISCUSSION TITLE
how to use rename

### DISCUSSION THREAD

**Discussion by laoshaw** [neoclide/coc.nvim#5089] [General]
when I hit <leader>rn I got: 
[coc.nvim' Error on notification "rename": Cannot rename symbole: symbol is not a supported kin(e.g. namespace,macro)

can I use it to replace any given string globally? what's its purposes? I'm on a c project.

**Comment by fannheyward** [comment]
> Cannot rename symbole: symbol is not a supported kind

`clangd` currently does not support renaming namespaces. https://github.com/clangd/clangd/issues/729

**Comment by laoshaw** [comment]
Thanks!
